app/block_address.go

```diff
@@ -41,6 +42,25 @@ func (bad BlockAddressesDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 				}
 			}
 		}
+
+		// check EIP-7702 authorisation list
+		for _, msg := range tx.GetMsgs() {
+			msgEthTx, ok := msg.(*evmtypes.MsgEthereumTx)
+			if ok {
+				ethTx := msgEthTx.AsTransaction()
+				if ethTx.SetCodeAuthorizations() != nil {
```

Comment:
when happened if the ethTx does not set CodeAuthorizations ? The tx execute will be failed?